virttest/utils_libvirt/libvirt_config.py

```diff
@@ -15,7 +15,8 @@
 
 
 def remove_key_in_conf(value_list, conf_type="libvirtd",
-                       remote_params=None, restart_libvirt=False):
+                       remote_params=None, restart_libvirt=False,
+                       mode='local'):
```

Comment:
From the function itself, 'local' removal happen no matter what situation it is.
so remote_params can be used to judge whether it should happen on remote host.
Not sure why we need introduce another parameter: 'mode'